manifest.md

```diff
@@ -133,16 +143,94 @@ Unlike the [image index](image-index.md), which contains information about a set
 
 ## Example of a SCRATCH config or layer descriptor
 
-Notice that the `mediaType` is subject to the usage or context, while the digest is specifically defined as `ScratchDigestSHA256`
+Notice that the `mediaType` is subject to the usage or context, while the digest is specifically defined as `ScratchDigestSHA256`.
+When the `ScratchDigestSHA256` is used, the media type SHOULD be set to `application/vnd.oci.scratch.v1+json` to differentiate the descriptor from one pointing to content.
 
 ```json,title=SCRATCH%20config&mediatype=application/vnd.oci.descriptor.v1%2Bjson
 {
-  "mediaType": "application/vnd.oci.example+json",
+  "mediaType": "application/vnd.oci.scratch.v1+json",
   "size": 2,
   "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a"
 }
 ```
 
+## Guidelines for Artifact Usage
+
+Content other than OCI container images MAY be packaged using the image manifest.
+When this is done, the `config.mediaType` value MUST be set to a value specific to the artifact type or the [scratch value](#example-of-a-scratch-config-or-layer-descriptor).
+If the `config.mediaType` is set to the scratch value, the `artifactType` must be defined.
+If the artifact does not need layers, a single layer SHOULD be included with a non-zero size.
+The suggested content for an unused layer is the [SCRATCH](#example-of-a-scratch-config-or-layer-descriptor) descriptor.
+
+Here is an example manifest for a typical artifact:
+
+```json,title=Artifact%20with%20config&mediatype=application/vnd.oci.image.manifest.v1%2Bjson
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
```

Comment:
should we add artifactType here..

Reply:
When config.mediaType is defined to something other than scratch, artifactType is optional.

Reply:
To be slightly prescriptive about best practices, should we say that non-image artifacts SHOULD use an `artifactType` with the scratch config?

Not a MUST.

Reply:
@AaronFriel I'm open to it, but in case there are other opinions, how about we make that a separate PR. Would you like to lead that?